docs/components/gantry/oneaxis.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Configure a One-Axis Gantry"
+linkTitle: "oneaxis"
+weight: 70
+type: "docs"
+description: "Configure a oneaxis gantry."
+tags: ["gantry", "components"]
+# SME: Rand
+---
+
+Configure a `oneaxis` gantry to integrate a one-axis gantry into your robot:
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and navigate to the **Create component** menu.
+Enter a name for your sensor, select the type `gantry`, and select the `oneaxis` model.
+
+Click **Create component**.
+Paste into the **Attributes** box:
+
+``` json
+"motor": <"your-motor-name">,
+"length_mm": <int>,
+"axis": {
+    "X": <int>,
+    "Y": <int>,
+    "Z": <int>
+}
+```
+
+![Creation of a one-axis gantry component in the Viam app config builder.](../img/oneaxis-ui-config.png)
+
+Edit and fill in the attributes as applicable.
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    // your motor config
```

Comment:
```suggestion
    // < Insert your motor config >
```
Or something similar to make it clear that this is a placeholder, not a label for what's below it.